### ISSUE TITLE
Upgrade Cosmos SDK to v0.45

### ISSUE THREAD

**Issue by youngjoon-lee** [medibloc/panacea-core#221]
Today, the cosmos-sdk v0.44.0 was released. 
https://github.com/cosmos/cosmos-sdk/releases/tag/v0.44.0

They are saying that all chains should upgrade cosmos-sdk to v0.44, regardless of whether their version is v0.42 or v0.43.

**Comment by youngjoon-lee** [comment]
Yeah. I haven't read the release note seriously, but it looks mandatory.

**Comment by youngjoon-lee** [comment]
I just promoted the target version from v0.44 to v0.46 because v0.46 is going to have a `x/nft` module which is a standard for IBC. 
From Cosmos Discord, the `x/nft` is already ready-to-use but will be released as v0.46.
https://discord.com/channels/669268347736686612/903696998874632282/951746679810428950
One of options for us is backporting the `x/nft` to panacea-core which is currently using cosmos-sdk v0.42.x, but it would be a tough work since v0.45~46 have a lot of breaking changes. Instead of backporting, it would be better to consider upgrading cosmos-sdk to v0.46 directly. Now, only v0.46.0-beta2 was released but it seems that v0.46.0 will be released in 19 days according to their milestone: https://github.com/cosmos/cosmos-sdk/milestone/46.
